<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogX509v3Extention</class>
 <widget class="QDialog" name="DialogX509v3Extention">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>X509v3 Extension chooser</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelextensionchooser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose or type an extension name :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxNameChooser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOKExtension">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelExtensionValue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>230</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Select value, edit it and click Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxExtensionValue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>251</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddValue">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>130</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditValue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelOutputValue">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Value : </string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxCritical">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Critical extension</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelExtensionNameOutput">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>extension_name=critical,</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonResetForm">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>280</y>
     <width>70</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Form</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAdd">
   <property name="geometry">
    <rect>
     <x>204</x>
     <y>280</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add and continue</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogX509v3Extention</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogX509v3Extention</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>380</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
